packages/terra-pills/src/private/_Pill.jsx

```diff
@@ -0,0 +1,313 @@
+          : undefined}
+        <span>{label}</span>
+      </div>
+    </Popup>
+  );
+
+  const renderRemoveButton = () => (
+    <button
+      {...removeButtonProps}
+      className={cx('pill-remove-button')}
+      tabIndex="-1"
+      type="button"
+      aria-labelledby={`remove-button-${id}`}
+      aria-hidden="true"
+    >
+      <span id={`remove-button-${id}`} className={cx('remove-button-label')}>{intl.formatMessage({ id: 'Terra.pills.label.delete' }, { pillLabelName: label })}</span>
+      <span className={cx('clear-icon')} />
+    </button>
+  );
+
+  const renderSelectablePill = () => {
+    const categoryLabel = labelCategory || '';
+    const visuallyHiddenTextHint = ` ${categoryLabel} ${label} ${pillInteractionHint}`;
+    return (
+      <>
+        <div
+          {...customProps}
+          {...pillProps}
+          {...pillButtonProps}
+          aria-expanded={pillInteraction.isSelectable ? true : undefined}
+          aria-haspopup={pillInteraction.isSelectable ? 'dialog' : undefined}
+          id={id}
+          className={pillClassNames}
+          ref={pillRef}
+          type="button"
+          aria-describedby={`interaction-hint-${id}`}
+          data-terra-pills-show-focus-styles
+          data-terra-pill
+        >
+          <span className={cx('pill-label')}>
+            {labelCategory
+              ? <span className={cx('pill-category')}>{`${labelCategory}: `}</span>
+              : undefined}
+            <span>{label}</span>
+          </span>
+        </div>
+        {pillInteraction.isRemovable && renderRemoveButton()}
+        {pillInteractionHint && <VisuallyHiddenText id={`interaction-hint-${id}`} text={visuallyHiddenTextHint} aria-hidden="true" />}
+        {isTruncated && renderTruncatedLabelPopup()}
+      </>
+    );
+  };
+
+  /* eslint-disable jsx-a11y/aria-role */
+  const renderBasicPill = () => {
+    const categoryLabel = labelCategory || '';
+    const visuallyHiddenTextHint = ` ${categoryLabel} ${label} ${pillInteractionHint}`;
+    return (
+      <>
+        <div
+          {...customProps}
+          {...pillProps}
+          id={id}
+          className={pillClassNames}
+          ref={pillRef}
+          role="text" // Prevent VoiceOver from announcing as "group" https://dequeuniversity.com/rules/axe/4.2/aria-text
+          aria-labelledby={`${pillInteractionHintID} interaction-hint-${id}`}
+          data-terra-pills-show-focus-styles
+          data-terra-pill
+        >
+          <label className={cx('pill-label')}>
+            {labelCategory
+              ? <span className={cx('pill-category')}>{`${labelCategory}: `}</span>
+              : undefined}
+            <span>{label}</span>
+          </label>
+        </div>
+        {pillInteraction.isRemovable && renderRemoveButton()}
+        {pillInteractionHint && <VisuallyHiddenText id={`interaction-hint-${id}`} text={visuallyHiddenTextHint} aria-hidden="true" />}
```

Comment:
This has `aria-hidden` set to true as well, doesn't that defeat the purpose of visually hidden text?

Reply:
This is removed now.